doc/index.rst

```diff
@@ -1,58 +1,41 @@
+Autocrypt support for mail agents
+=================================
 
-Autocrypt Python Reference tool and implementation
-==================================================
+py-autocrypt provides a :doc:`cmdline tool <cmdline>` and a :doc:`Python API <api>`, to help mail agents evolve `Autocrypt <https://autocrypt.org>`_ support.
 
-.. note::
+See :doc:`install` for getting started.
 
-    The py-autocrypt tool is as much in development
-    as the spec itself.  Until we have a 1.0 release
-    everything is subject to change.
+Aims and goals
+--------------
 
-.. toctree::
+- `Autocrypt Level 1 compliant functionality
```

Comment:
So, "Autocrypt Level 1 compliant functionality" is a goal, but doesn't say about the status.